src/FSharp.ProjectTemplate/FSharp.ProjectTemplate.fsproj

```diff
@@ -19,19 +19,19 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <Tailcalls>false</Tailcalls>
-    <OutputPath>..\..\bin</OutputPath>
+    <OutputPath>.\bin\Debug</OutputPath>
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <WarningLevel>3</WarningLevel>
-    <DocumentationFile>..\..\bin\FSharp.ProjectTemplate.xml</DocumentationFile>
+    <DocumentationFile>.\bin\Debug\FSharp.ProjectTemplate.xml</DocumentationFile>
```

Comment:
?
The project is named `FSharp.ProjectTemplate` so I assume the xml docs should have the same name. (Unless I'm completely misunderstanding you question here)